lib/ruy/conditions/in_cyclic_order.rb

```diff
@@ -1,20 +1,28 @@
 module Ruy
   module Conditions
+
+    # Expects that a value is contained in a cyclic order
+    #
     class InCyclicOrder < Condition
-      attr_reader :from, :to, :attr
+      attr_reader :from, :to, :key
 
-      def initialize(from, to, *attrs)
+      # @param from left bound
+      # @param to right bound
+      # @param key
+      # @example check that :key is between a cycle defined between 100 and -100
```

Comment:
THe same as for between a range. I believe that it should be: between 100 and -100 or included in a cycle between 100 and -100.